actionpack/lib/action_dispatch/system_testing/test_helpers/warning_http_verb.rb

```diff
@@ -0,0 +1,17 @@
+module ActionDispatch
+  module SystemTesting
+    module TestHelpers
+      module WarningHttpVerb # :nodoc:
+        MESSAGE = "This is SystemTesting. You should use `visit` or any other Capybara API."
```

Comment:
it would be nice to have a frozen string here:
```ruby
MESSAGE = "This is SystemTesting. You should use `visit` or any other Capybara API.".freeze
# this is just a personal style guide, but this can also be a private constant
# private_constant :MESSAGE
```